stack.yaml

```diff
@@ -1,5 +1,6 @@
-resolver: lts-12.9
+resolver: lts-13.20
```

Comment:
`lts-13.21` is already out! :slightly_smiling_face:

Reply:
Nice! I’ll change it